thunder/dynamo/report.py

```diff
@@ -1235,22 +1248,20 @@ def thunderfx_benchmark_report_from_splits(
 
 def thunderfx_benchmark_report(
     fn: Callable,
-    *args,
     folder_path: str | PathLike,
-    thunder_compile_kwargs: dict = None,
     check_torch_runnablility: bool = True,
     time_rtol=0.5,
     time_atol=0.0,
     memory_usage_rtol=0.5,
     memory_usage_atol=0.0,
     compare_fusion: bool = False,
     stream: TextIO = sys.stdout,
-    **kwargs,
+    **compile_kwargs,
 ):
     """
     A utility function that analyzes the runnability and performance benchmarks of each FX graph.
 
-    1. Checks if the callable can be executed with `torch.compile`.
+    1. Checks if the callable can be executed with `torch.compile` when `check_torch_runnablility` is `True`.
```

Comment:
Nit: in general it's nice to start with the condition upfront, like:

"If `check_torch_runnability` is `True`, ..."